src/Symfony/Component/Debug/Exception/FlattenException.php

```diff
@@ -90,79 +90,125 @@ public function getStatusCode()
         return $this->statusCode;
     }
 
+    /**
+     * @return $this
+     */
     public function setStatusCode($code)
     {
         $this->statusCode = $code;
+
+        return $this;
```

Comment:
Do we really need fluent interface here?

Reply:
We don't *need* it. It still makes it easier to use the class.